tools/build/configure.py

```diff
@@ -1015,6 +1018,12 @@ def build(
                     },
                 )
                 build(entry.object_path, [entry.object_path.with_suffix("")], "bin")
+            elif seg.type == "pm_sbn":
+                sbn_path = entry.object_path.with_suffix("")
+                build(sbn_path, entry.src_paths, "pm_sbn") # could have non-yaml inputs be implicit
```

Comment:
The non-yaml inputs _should_ be marked as implicit

Reply:
I haven'tr eally looked into the build system all that much - in fact I've never worked with ninja before- so I don't quite understand what this means.

Reply:
if I understand alex's message correctly, I think he's referring to one of the optional arguments of build(), which is a list of implicit dependencies...or outputs? can't remember. meowp

Reply:
Yeah, so implicit is a kwarg to this function that takes a list of paths that are to be considered dependencies of this build step but aren't actually included in the build command (in which case they'd just be normal dependencies). Applied to this case, the idea is that the files that are used to produce the SBN besides the yaml file (BGM files, MSEQ files, etc) should be given as `implicit`. See https://ninja-build.org/manual.html#ref_dependencies for more info

Reply:
Decision on Discord was to implement this using depfiles for all asset types in a future PR. See #1098